client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-item.jsx

```diff
@@ -29,9 +29,41 @@ const EmailForwardingItem = React.createClass( {
 			this.recordEvent( 'deleteClick', domain, mailbox, forward_address, ! Boolean( error ) );
 
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'Failed to delete email forwarding record. Please try again or contact customer support if error persists.' ) );
 			} else {
-				this.props.successNotice( this.translate( 'Yay, %(email)s has been successfully deleted!', { args: { email: email } } ) );
+				notices.success(
+					this.translate( 'Yay, %(email)s has been successfully deleted!', {
```

Comment:
Maybe we should say something along the lines of "Yay, Email forwarding for %(email)s has been successfully deleted. Further emails will no longer be forwarded to this address".

A message "yourinbox@example.com has been deleted" might cause little a scare. :)